test/data/a.txt

```diff
@@ -0,0 +1 @@
+xyz1
```

Comment:
Add general strings to make a.txt file pass uniitests( previously it was empty so lead to failure.)